apps/payments/next/app/[offeringId]/checkout/page.tsx

```diff
@@ -0,0 +1,47 @@
+import { getContentfulContent, getCartData } from '../../lib/apiClient';
+import { PurchaseDetails, TermsAndPrivacy } from '@fxa/payments/ui/server';
+
+interface CheckoutParams {
+  offeringId: string;
+}
+
+export default async function Index({ params }: { params: CheckoutParams }) {
+  // TODO - Fetch Cart ID from cookie
+  // https://nextjs.org/docs/app/api-reference/functions/cookies
+  const cartId = 'cart-uuid';
+  // TODO - Fetch locale from params
+  // Possible solution could be as link below
+  // https://nextjs.org/docs/app/building-your-application/routing/internationalization
+  const locale = 'en-US';
+
+  const contentfulData = getContentfulContent(params.offeringId, locale);
+  const cartData = getCartData(cartId);
+  const [contentful, cart] = await Promise.all([contentfulData, cartData]);
+
+  return (
+    <>
+      <h1 className="page-title-container">Under Construction</h1>
+      <section className="payment-panel" aria-label="Purchase details">
```

Comment:
Should this `section` be an `aside`?
cc: @sardesam

Reply:
I initially had it like this as well. Reading through these docs though, [W3 - Complementary Landmark](https://www.w3.org/WAI/ARIA/apg/patterns/landmarks/examples/complementary.html), it doesn't seem like Purchase Details is an `<aside>`/complementary piece of the page, but still part of the main content, which is why I left it under `<main>` and split it up into `<section>`'s.

That was just my interpretation though. Happy to keep discussing.

Reply:
Noted! I asked as it would appear to be an `aside` in tablet/desktop view, but did not seem correct in mobile view, but wanted to ask :) Thanks!